File: IAIso-v5.0/core/iaiso-java/iaiso-policy/src/main/java/io/iaiso/policy/MeanAggregator.java
package io.iaiso.policy;

import java.util.Map;

/** Arithmetic mean of per-execution pressures. */
public final class MeanAggregator implements Aggregator {
    @Override public String name() { return "mean"; }
    @Override public double aggregate(Map<String, Double> p) {
        if (p.isEmpty()) return 0.0;
        double total = 0.0;
        for (Double v : p.values()) total += v;
        return total / p.size();
    }
}

File: IAIso-v5.0/core/iaiso-php/src/Policy/MeanAggregator.php
<?php

declare(strict_types=1);

namespace IAIso\Policy;

/** Arithmetic mean of per-execution pressures. */
final class MeanAggregator implements Aggregator
{
    public function name(): string { return 'mean'; }

    public function aggregate(array $pressures): float
    {
        if (count($pressures) === 0) return 0.0;
        return array_sum($pressures) / count($pressures);
    }
}
